themes/beaver/layouts/page/index.html

```diff
@@ -24,9 +24,10 @@
 
    document.head.appendChild(link);
 }
-
+{{- $dynamicCSS := resources.Get "css/dynamic-404-590.css" | resources.ExecuteAsTemplate "css/dynamic.css" . }}

```

Comment:
Are we using start cta only on those 2 pages? Other pages do not have such block?

Reply:
only 2 pages